### ISSUE TITLE
intl 0.18.0 is not supported

### ISSUE THREAD

**Issue by Hesamedin** [andresaraujo/timeago.dart#208]
I have intl 0.17.0 in my pubspec.yaml file. I upgraded the intl to 0.18.0 and this error displays:
```
Because timeago 3.3.0 depends on intl ^0.17.0 and no versions of timeago match >3.3.0 <4.0.0, timeago ^3.3.0 requires intl ^0.17.0.
So, because ssm depends on both intl ^0.18.0 and timeago ^3.3.0, version solving failed.
pub get failed (1; So, because ssm depends on both intl ^0.18.0 and timeago ^3.3.0, version solving failed.)
```

So, I need to keep the intl to 0.17.0 until you support it. Thank you.

**Comment by thorizer** [comment]
this is the only package that requires an old version of intl , and I'm using a lot

**Comment by andresaraujo** [comment]
Fixed in #203
